test/integration/targets/vmware_guest/tasks/create_multi_disks_d1_c1_f0.yml

```diff
@@ -0,0 +1,1212 @@
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+    state: poweredoff
+    folder: "{{ item|dirname }}"
+  with_items: "{{ vmlist['json'] }}"
+  register: clone_d1_c1_f0
+
+- debug: var=clone_d1_c1_f0
+
+- name: assert that changes were made
+  assert:
+    that:
+        - "clone_d1_c1_f0.results|map(attribute='changed')|unique|list == [true]"
+
+- name: create new VMs with 15 disks again
+  vmware_guest:
+    validate_certs: False
+    hostname: "{{ vcsim }}"
+    username: "{{ vcsim_instance['json']['username'] }}"
+    password: "{{ vcsim_instance['json']['password'] }}"
+    name: "{{ 'multi_disks_15_' + item|basename }}"
+    guest_id: centos64Guest
+    datacenter: "{{ (item|basename).split('_')[0] }}"
+    hardware:
+        num_cpus: 1
+        memory_mb: 512
+    disk:
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+        - size: 0gb
+          type: thin
+          autoselect_datastore: True
+    state: poweredoff
+    folder: "{{ item|dirname }}"
+  with_items: "{{ vmlist['json'] }}"
+  register: clone_d1_c1_f0
+
+- debug: var=clone_d1_c1_f0
+
+- name: assert that changes were made
```

Comment:
same as earlier comment, about changes *not* being made, this looks to be in a couple of places.